plugins/org.jkiss.dbeaver.ext.mysql/src/org/jkiss/dbeaver/ext/mysql/model/MySQLUser.java

```diff
@@ -303,4 +309,30 @@ public DBSObject refreshObject(@NotNull DBRProgressMonitor monitor) throws DBExc
         grants = null;
         return this;
     }
+
+    @Nullable
+    public String getPersistedUserName() {
+        return persistedUserName;
+    }
+
+    public void setPersistedUserName(@Nullable String persistedUserName) {
+        this.persistedUserName = persistedUserName;
+    }
+
+    @Nullable
+    public String getPersistedHost() {
+        return persistedHost;
+    }
+
+    public void setPersistedHost(@Nullable String persistedHost) {
+        this.persistedHost = persistedHost;
+    }
+
+    public String getPersistedFullName() {
+        return getFullName(persistedUserName, persistedHost);
+    }
+
+    private static String getFullName(@Nullable String userName, @Nullable String host) {
+        return "'" + userName + "'@'" + host + "'";
```

Comment:
Does this makes sense to allow nullable values here? In the worst case, it will result in `'null'@'null'`

Reply:
It totally makes sence since this is exactly what's going to happen when we create a new user. The fields `persistedUserName` and `persistedHost` are null in this case. Moreover, IMHO `@Nullable`'s meaning is to indicate that is _safe_ to pass null as an argument.